docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Teleport to create host users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+The following utilities should be available in the PATH:
+- useradd
+- userdel
+- usermod
+- groupadd
+- getent
+- visudo
+
```

Comment:
```suggestion
(!docs/pages/includes/tctl.mdx!)
```
We should include the `tctl.mdx` partial since the instructions in this guide use dynamic resources.